controllers/storagecluster/storageconsumer.go

```diff
@@ -109,7 +112,17 @@ func (s *storageConsumer) ensureDeleted(_ *StorageClusterReconciler, _ *ocsv1.St
 	return ctrl.Result{}, nil
 }
 
-func getLocalStorageClassNames(ctx context.Context, kubeClient client.Client, storageCluster *ocsv1.StorageCluster) (
+func getExternalClassesBlaclistSelector(log logr.Logger) labels.Selector {
+	blackListRequirement, err := labels.NewRequirement(util.ExternalClassLabelKey, selection.NotEquals, []string{"true"})
+	if err != nil {
+		log.Error(err, "failed to create label requirement for external classes")
```

Comment:
- Error shouldn't be logged without returning them and effecting the result of the reconcile.
- What kind of error can even be returned here? wouldn't it be safe to ignore it all of the requirement information is hardcoded?

Reply:
Made it info

Reply:
added a panic as discussed offline, as it will be always a static error